### ISSUE TITLE
TieFix Translation issues

### ISSUE THREAD

**Issue by Amirhan-Taipovjan-Greatest-I** [j-tai/TieFix#16]
First of all, I think that It would be nice if the descriptions of bugs were in the tooltips... because if I want to translate this mod, for example, into Tatar Language... I have a BIG problem with text fitting:
![image](https://user-images.githubusercontent.com/51203385/220077197-976a14e3-093d-43d6-a24a-b2f353d40c9e.png)

I'm not so sure about this moment, because I tested v.1.11.0, but I think that nothing changed in newest version:
Also, as I can see, there's missing "No", "Yes", "Reset" and "Save & Quit" strings in the file "en_us.json"... 
You can use an Vanilla-ish translations for "Yes" and "Reset", but "No" and "Save & Quit" strings still need to be added in source translation file.

**Comment by j-tai** [comment]
Thanks for contributing :)

Yes, this is something that has frustrated me as well. I had to spend some effort to fit the English text in that space, though this isn't an ideal solution, and I recognize that some languages may take more space.

This mod currently uses [Cloth Config](https://github.com/shedaniel/cloth-config) to generate the configuration screen. I'll have to look into whether Cloth Config can wrap the text. The reason I didn't put the description in the tooltips is because that makes it much slower to find the specific fix you are looking for. People will usually look for options based on the description, not the bug ID.

To translate the No/Yes/Reset/etc. texts, they should be translated in Cloth Config. See here: https://github.com/shedaniel/cloth-config/tree/v8/common/src/main/resources/assets/cloth-config2/lang

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
> This mod currently uses [Cloth Config](https://github.com/shedaniel/cloth-config) to generate the configuration screen. I'll have to look into whether Cloth Config can wrap the text. The reason I didn't put the description in the tooltips is because that makes it much slower to find the specific fix you are looking for. People will usually look for options based on the description, not the bug ID.

Yeah, I can agree with You... I just suggested the way that I see as the solution for this issue.


> To translate the No/Yes/Reset/etc. texts, they should be translated in Cloth Config. See here: https://github.com/shedaniel/cloth-config/tree/v8/common/src/main/resources/assets/cloth-config2/lang

Unfortunately, this project seems to be abandoned more than 1 month ago... because I saw submission of other translation files and their updates...

**Comment by j-tai** [comment]
Okay, I took a look at Cloth Config's code, and it appears there is no way to make Cloth Config wrap the text automatically.

I could try to add line wrapping on TieFix's side, though...

For now, you can continue translating if you like, and I'll figure out how to make the text fit.

> Unfortunately, this project seems to be abandoned more than 1 month ago... because I saw submission of other translation files and their updates...

I would recommend that you submit a PR anyway. It's likely that the author(s) will come back and update it once the next Minecraft version releases.

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
> For now, you can continue translating if you like, and I'll figure out how to make the text fit.

I think I should wait for new patch where You figure out how to make text fit, I guess...


> I would recommend that you submit a PR anyway. It's likely that the author(s) will come back and update it once the next Minecraft version releases.

Yeah, I submitted it right now... I hope, that I and Kazakh colleague won't be ignored =)

**Comment by j-tai** [comment]
If you're done translating, you can just submit the PR and ignore the text overlapping the buttons. I will see if I can figure out a way to wrap the text. When I do that, you won't need to update the translation again.

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
#18
